toolkits/global/packages/global-expander/HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # History
 
+## 2.1.0 (2020-09-21)
+    * Use isOpen as source of truth
```

Comment:
Can this elaborate a little bit more to describe what it is being used as the source of truth of?

Reply:
Sure will do